HybridCRT.props

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is only included in one other file, `Cpp.Build.props`. Instead of cluttering the root directory of the project with build rules, can we just merge it inside `Cpp.Build.props`?